spec/api/v1/schemas/hosts.rb

```diff
@@ -1,23 +1,57 @@
 # frozen_string_literal: true
 
+require './spec/api/v1/schemas/util'
+
 module Api
   module V1
     module Schemas
       module Hosts
+        extend Api::V1::Schemas::Util
+
         HOST = {
           type: 'object',
-          required: %w[name account_id],
```

Comment:
account_id is an odd thing to show users IMO. Everything in our API is scoped to the current account.

Reply:
Agreed, we should ask @Victoremepunto to confirm this is not used in any tests QA runs.

Reply:
Doesn't rings a bell, I don't think this is going to impact us. thanks for the heads up.